draft-ietf-rats-architecture.md

```diff
@@ -296,56 +296,62 @@ each router is an Attester and the main router is the Lead Attester.
 {{composite}} depicts the conceptual data flow for a Composite Device.
 
 ~~~~
-                   .-----------------------------.
-                   |           Verifier          |
-                   '-----------------------------'
-                                   ^
-                                   |
-                                   | Composite
-                                   | Evidence
```

Comment:
I didn't come up with a clear definition of 'Composite Evidence'. Besides, according to Eric's new draft, the Composite Evidence is not only applicable to the Composite Device. So I change here to 'Evidence of Composite Device'. Maybe we can create an issue about defining Composite Evidence.